chromium_src/chrome/browser/policy/profile_policy_connector.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_POLICY_PROFILE_POLICY_CONNECTOR_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_POLICY_PROFILE_POLICY_CONNECTOR_H_
+
+#include "base/timer/timer.h"
+
+// Add virtual to Init and Shutdown
+#define Init(user, schema_registry, configuration_policy_provider,         \
+             policy_store, browser_policy_connector, force_immediate_load) \
+  virtual Init(user, schema_registry, configuration_policy_provider,       \
+               policy_store, browser_policy_connector, force_immediate_load)
+#define Shutdown()    \
+  virtual Shutdown(); \
+  friend class BraveProfilePolicyConnector
+#define final
```

Comment:
You can't do this, it's undefined behavior to redefine keywords. You will have to patch

Reply:
reverted to the patch way, I don't think there's any other way around this.